mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping

```diff
@@ -0,0 +1,53 @@
+CLASS net/minecraft/class_8985 net/minecraft/client/render/debug/BreezeDebugRenderer
+	FIELD field_47465 PINK I
+	FIELD field_47466 LIGHT_BLUE I
+	FIELD field_47467 GREEN I
+	FIELD field_47468 ORANGE I
+	FIELD field_47469 RED I
+	FIELD field_47472 client Lnet/minecraft/class_310;
+	FIELD field_47473 breezeDebugInfoList Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/class_310;)V
+		ARG 1 client
+	METHOD method_55255 clearMap ()V
+	METHOD method_55256 draw (ILorg/joml/Matrix4f;DDDLnet/minecraft/class_4588;Lnet/minecraft/class_243;FI)V
+		ARG 0 vertex
+		ARG 1 matrix4f
+		ARG 8 vertexConsumer
+		ARG 9 entityBox
+	METHOD method_55257 (Lnet/minecraft/class_1297;)Lnet/minecraft/class_243;
+		ARG 1 entity
+	METHOD method_55258 (Lnet/minecraft/class_8949;)Z
+		ARG 0 predicate
+	METHOD method_55259 renderOutlines (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDD)V
```

Comment:
```suggestion
	METHOD method_55259 render (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDD)V
```

Mojang forgot to override `DebugRenderer.Renderer`; if they did, this method would've been named "render".